### PR TITLE
Added test cases for wc_curve25519_export_key_xx

### PR DESCRIPTION
**Pull request by TakayukiMatsuo**
Added following test cases to tests/api.c 
- test_wc_curve25519_export_key_raw()
- test_wc_curve25519_export_key_raw_ex()

### CODE REVIEW

File: tests/api.c [hunk 1]
```diff
@@ -15184,6 +15184,535 @@ static int test_wc_curve25519_size (void)
...
+        fflush( stdout );
+        wc_curve25519_free(&key);
+        wc_FreeRng(&rng);
+        return  1;
+    }
+        
+    prvkSz = CURVE25519_KEYSIZE;
+    pubkSz = CURVE25519_KEYSIZE;
+    
+    if(BAD_FUNC_ARG != wc_curve25519_export_key_raw_ex( &key,privateKey,       NULL,publicKey, &pubkSz,EC25519_LITTLE_ENDIAN)){
```
Comment (cconlon) [inline tests/api.c:15465]: This line goes over 80 characters, please wrap to next line.

File: tests/api.c [hunk 2]
```diff
@@ -15184,6 +15184,535 @@ static int test_wc_curve25519_size (void)
...
+        fflush( stdout );
+        wc_curve25519_free(&key);
+        wc_FreeRng(&rng);
+        return  1;
+    }
+    
+    prvkSz = CURVE25519_KEYSIZE;
+    pubkSz = CURVE25519_KEYSIZE;
+
+    if(BAD_FUNC_ARG != wc_curve25519_export_key_raw_ex( &key, privateKey,       NULL, publicKey, &pubkSz, EC25519_BIG_ENDIAN)){
```
Comment (cconlon) [inline tests/api.c:15529]: This line goes over 80 characters, please wrap to next line.

**Comment by TakayukiMatsuo** [comment]
I have resolved above three points you requested in a new branch, but could not in the original  branch-1. Would you mind my opening a new  pull request and closing this request?

**Comment by TakayukiMatsuo** [comment]
Closed since the new pull request #3062 was accepted.

**Pull request closed without merging**